app/views/articles/_v2_form.html.erb

```diff
@@ -3,8 +3,29 @@
     padding-top: 0 !important;
   }
 
-  .crayons-footer,
-  .crayons-header {
+  .crayons-header,
+  .crayons-header * {
+    visibility: hidden;
+    position: absolute;
+    left: -9999em;
+    top: -9999em;
+    pointer-events: none;
+    opacity: 0;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+  }
+
+  .crayons-header .site-logo,
+  .crayons-header .site-logo * {
+    visibility: visible;
+    position: static;
+    pointer-events: auto;
+    opacity: 1;
+    clip: auto;
+    overflow: visible;
```

Comment:
This is probably an overkill, but I wanna be sure it will work :D

Reply:
probably worth explaining a bit:

previously we applied `display: none;` to `.crayons-header` - which basically hides the header and everything inside it (including logo). so this piece of code is meant to replace the `display: none;` styling in a way.. BUT without affecting the logo... 

![](https://media.giphy.com/media/13FrpeVH09Zrb2/source.gif)

Reply:
refactor (non-blocking): This definitely works, but is probably overkill like you said @ludwiczakpawel.

Given that the children of `.crayons-header` are absolutely positioned off the screen and `pointer-events` are set to `none`, you probably only need these properties:

```css
.crayons-header,
  .crayons-header * {
    position: absolute;
    left: -9999em;
    top: -9999em;
    pointer-events: none;
}
```

and then the reset like so.

```
 .crayons-header .site-logo,
 .crayons-header .site-logo * {
  position: static;
}
```

Reply:
We need a bit more here so users can't access it with keyboard navigation... But yea, definitely can be simplified a little. I like to over-engineer things :D